kitsune/sumo/static/sumo/scss/layout/_profile.scss

```diff
@@ -13,5 +13,5 @@
 }
 
 .avatar {
-  padding-top: p.$spacing-lg;
+  margin-top: p.$spacing-lg;
 }
```

Comment:
The root of the problem here is this selector is overly broad, something like `#edit-profile .avatar` will only select the avatar on the edit profile page, without selecting the one in the navbar as well.

I'd also suggest using `$spacing-sm` as that's the same spacing we have between labels and fields below.